Lib/Protocols/IdFTPServer.pas

```diff
@@ -6059,6 +6172,34 @@ procedure TIdFTPServer.DoOnDataPortBeforeBind(ASender: TIdFTPServerContext);
   end;
 end;
 
+function TIdFTPServer.GetPathSeperator : String;
+begin
+  Result := '/';
+  case FPathProcessing of
+    ftppDOS : Result := '\';
+    ftpOSDependent :
+      begin
+        if (GOSType = otWindows) then
+        begin
+          Result := '\';
+        end;
+      end;
+  end;
+end;
+
+function TIdFTPServer.FTPIsCaseSensitive: Boolean;
+begin
+  case FPathProcessing of

```

Comment:
What about `ftppCustom`? Do we need a way for the user to indicate whether they are case-sensitive or not?